src/ERC4973.sol

```diff
@@ -77,4 +79,18 @@ abstract contract ERC4973 is ERC165, IERC721Metadata, IERC4973 {
 
     emit Revoke(owner, tokenId);
   }
+
+  function _mintWithPermission(
+    address from,
+    address to,
```

Comment:
Actually, I'm now realizing that passing `address to` as I've outlined in #7 introduces a security vector where two addresses (`address from` and `msg.sender`) could non-consensually send a token to `address to`.

Hence, I suggest to entirely remove `address to` as a function parameter for `_mintWithPermission` and in its function body replace it with `msg.sender´.

In this way; 

-`msg.sender´ can never control `address from` without first asking from's key controller for a signature
and `address from` can never directly mint to `address to` as only the controller of `address to`'s private key can send a transaction to call `_mintWithPermission` (that internally uses `msg.sender`).

Reply:
Good point. I agree as well.

Reply:
@rahulrumalla you resolved this item but didn't make the suggested change. Did you send the change?